### PR TITLE
Add Apptentive as a package to flutter glovebox.

### PR DESCRIPTION
**Pull request by aanelson**
Adds Apptentive to the flutter glovebox

The api surface is a single method as we are only using the engage event and the setup is done during application startup before flutter has initialized.

### CODE REVIEW

File: packages/apptentive_flutter/LICENSE [hunk 1]
```diff
@@ -0,0 +1,12 @@
+Copyright 2019-2020 eBay Inc.
```
Comment (aanelson) [inline packages/apptentive_flutter/LICENSE:1]: copied from golden_toolbox

**Comment by coreysprague** [comment]
@lmckenzie @mZadorskii @aanelson -- my two cents is that we should request a new repo for this. thoughts?

**Comment by mZadorskii** [comment]
We should, yes.

**Comment by aanelson** [comment]
Flutter glovebox is setup like flutter/plugin repo, is there any reason we would have the same architecture but only limit it to a single package?  We should flatten this repo to just be golden_toolbox then I would imagine as well.

**Comment by coreysprague** [comment]
my personal pref is to lean towards the latter, but there are differing opinions.

That being said, Larry, max and I previously discussed apptentive, and there's a possibility of transferring ownership over to apptentive at some point, so it might make sense to not bundle.

**Comment by lmckenzie** [comment]
The intention of the flutter_glove_box is to bring together several related but distinctly different packages. It is a work in progress so we will see how it grows when we add the next package. Apptentive does not belong here as it is related to a 3rd party product. @aanelson please make an internal repo for the apptentive plugin and I can help with the process of getting an external repo added. I a closing this PR.

**Pull request closed without merging**